lib/iris/experimental/ugrid/mesh.py

```diff
@@ -164,14 +170,14 @@ def __init__(
             Either ``0`` or ``1``. Default is ``0``. Denotes whether
             :attr:`indices` uses 0-based or 1-based indexing (allows support
             for Fortran and legacy NetCDF files).
-        * src_dim (int):
-            Either ``0`` or ``1``. Default is ``0``. Denotes which dimension
-            of :attr:`indices` varies over the :attr:`src_location`\\ s (the
-            alternate dimension therefore varying within individual
-            :attr:`src_location`\\ s). (This parameter allows support for fastest varying index being
+        * location_axis (int):
+            Either ``0`` or ``1``. Default is ``0``. Denotes which axis
+            of :attr:`indices` varies over the :attr:`location` elements (the
+            alternate axis therefore varying over :attr:`connected` elements).
+            (This parameter allows support for fastest varying index being
             either first or last).
             E.g. for ``face_node_connectivity``, for 10 faces:
-            ``indices.shape[src_dim] = 10``.
+            ``indices.shape[location_axis] = 10``.
```

Comment:
I think "==" would be clearer than "=" here. 
I.E. "it is...", not "set it to ..."

Reply:
7471929b585df6ff8718a75b2224b1027a5331d2